source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,51 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-04-21
+
+### Highlights
+
+#### Apps Framework (Developer Preview)
+ - This feature is available on Mobile Apps when Apps Framework becomes available on Mattermost v5.35.0 and the proxy plugin is loaded on an instance. The launch for the Developer Preview of the Apps Framework is scheduled for April 29th.
+
+#### Files Search
+ - Users can now search file attachments by name and document contents. This feature is available on Mobile Apps in a future release.
```

Comment:
Searching in Mattermost now finds both relevant messages and files in your team's conversation history. Search will return results for attachments that match the file name or contain matching text content within supported document types. [Learn more](https://mattermost.com/blog/file-search/)